specs/signing-scheme.md

```diff
@@ -51,8 +51,8 @@ This attribute dictates the rest of signature schema - the set of signed and uns
 
 Both `notary.x509` and `notary.x509.signingAuthority` signing schemes use the similar signature schema (set of signed and unsigned attributes) with the following differences.
 
-* `notary.x509` can ONLY use a countersignature (timestamp) from a trusted source to determine authentic signing time.
-This is supported through the OPTIONAL *Timestamp Signature* unsigned attribute in the signature envelope. Currently Notary Project uses a [RFC3161][ietf-rfc3161] compliant TSA signature for this purpose. When NOT provided, every certificate MUST be valid at the time of verification.
+* `notary.x509` does not have the `authentic signing time` concept. It can ONLY use a countersignature (timestamp) to prove that the signature was created on or before the time point been stamped by the `Timestamp Authority (TSA)`.
```

Comment:
```suggestion
* `notary.x509` does not have the `authentic signing time` concept. It can ONLY use a countersignature (timestamp) to prove that the signature was created on or before the time point that has been stamped by the `Timestamp Authority (TSA)`.
```